app/api/__init__.py

```diff
@@ -315,6 +315,8 @@
 api.route(EventRelationship, 'event_stripe_authorization', '/events/<int:id>/relationships/stripe-authorization',
           '/events/<identifier>/relationships/stripe-authorization')
 # Events -> roles:
+api.route(EventRelationship, 'event_owner', '/events/<int:id>/relationships/owner',
+          '/events/<identifier>/relationships/owner')
```

Comment:
what about the corresponding user relationship?

Reply:
@CosmicCoder96 I've added owner to event relationship -
`api.route(UserRelationship, 'user_owner_event', '/users/<int:id>/relationships/owner-events')`